esx_service/vmci/vmci_client.c

```diff
@@ -349,5 +366,22 @@ Vmci_GetReply(int port, const char* json_request, const char* be_name,
    req.mlen = strnlen(json_request, MAXBUF) + 1;
    req.msg = json_request;
 
+   ans->errBuf = malloc(512);
```

Comment:
if it's a fixed size, why not simply define it as char [BUF_SIZE] instead of trying to malloc and free ?